src/Microsoft.Framework.ApplicationHost2/Program.cs

```diff
@@ -29,6 +31,13 @@ public Program(IAssemblyLoaderContainer container, IApplicationEnvironment envir
 
         public Task<int> Main(string[] args)
         {
+#if DEBUG && ASPNET50
+            if(args.Length > 0 && string.Equals(args[0], "dbg", StringComparison.OrdinalIgnoreCase))
```

Comment:
Yeah... --debug doesn't launch the debugger which is kind of annoying. Perhaps we can add "--debug-launch" that causes the VS JIT Debugger dialog to pop up. It's really useful to be able to just click your current VS instance and click OK rather than have to attach to process.

I'll remove this for now.